packages/same-structure/README.md

```diff
@@ -0,0 +1 @@
+# Same Structure
```

Comment:
needs more

Reply:
I'll make an issue for this so that we can decouple the need to explain more about same structure from the need to make the package.

Reply:
See #492